field_test.go

```diff
@@ -129,3 +129,14 @@ func TestNestField(t *testing.T) {
 	nest := Nest("foo", String("name", "phil"), Int("age", 42))
 	assertCanBeReused(t, nest)
 }
+
+func TestStackField(t *testing.T) {
+	enc := newJSONEncoder()
+	defer enc.Free()
+
+	Stack().addTo(enc)
+	output := string(enc.bytes)
+
+	assert.Equal(t, `"stacktrace":`, output[:13], "Stacktrace added under an unexpected key.")
```

Comment:
might want to do `strings.HasPrefix` so the test doesn't panic if `output` is smaller than expected, and make it a `require` so that it doesn't run the below check if the string doesn't start with `"stacktrace":`